proposals/evaluation/uc-3-inheritance.md

```diff
@@ -0,0 +1,183 @@
+  acp:applyMembers </acp/research#p1> . # applies the policy to all resources contained by </weekly-status/>
+```
+
+Todo:  what is the ACR for 
+`</weekly-status/2021-04-28/report.md>` and what information is in it?
+   
+### WAC
+
+Bob and Alice are members of the research group `</groups/research#g1>`:
+
+```turtle
+# Resource: </groups/research>
+<#g1>
+  a vcard:Group ;
+  vcard:hasMember ex:Bob ;
+  vcard:hasMember ex:Alice .
+```
+
+The acl enabling read access to all resources contained by `</weekly-status/>` for all members of group `</groups/research#g1>` is:
+
+```turtle
+# Resource: </weekly-status/.acl>
+<#authorization>
+  a acl:Authorization ;
+  acl:agentGroup </groups/research#g1> ;
+  acl:default <.> ;
+  acl:mode acl:Read .
+```
+
+
+## 2. Changing permissions to a subcollection
+
+Bob wants to give Carol read/write access but only to the collection `</weekly-status/2021-04-28/>`.
+
+(What example from the UCR does this correspond to best?)
+
+### ACP 
+
+Carol is part of the agent matcher `</acp/research#m2>`:
+
+```turtle
+# Resource: </acp/research>
+<#m2>
+  a acp:AgentMatcher ;
+  acp:agent ex:Carol .
+```
+
+The research policy `</acp/research#p2>` gives read access to all agents matched by `</acp/research#m2>`:
+
+```turtle
+# Resource: </acp/research>
+<#p2>
+  a acp:Policy ;
+  acp:anyOf <#m2> ;
+  acp:allow acl:Read, acl:Write .
+```
+
+The access control `</weekly-status/2021-04-28/.acp#ac1>` applies to all resources contained by `</weekly-status/2021-04-28/>` and via policy `</acp/research#p2>` enables read access for all agents matched by `</acp/research#m2>`:
+
+```turtle
+# Resource: </weekly-status/2021-04-28/.acp>
+<#ac1>
+  a acp:AccessControl ;
+  acp:apply </acp/research#p1>, </acp/research#p2> ;
+  acp:applyMembers </acp/research#p1>, </acp/research#p2> . # applies the policy to all resources contained by </weekly-status/2021-04-28/>
+```
+
+### WAC
+
+To change the access rules to the `</weekly-status/2021-04-28/>` collection Bob must edit `</weekly-status/2021-04-28/.acl>`. But doing so will disable the default access control rule located in `</weekly-status/.acl>` . So the content of the PUT must contain the default rule and the new ones.
+
+```Turtle
+# Resource: </weekly-status/2021-04-28/.acl>
+<#authorization>
+  a acl:Authorization ;
+  acl:agentGroup </groups/research#g1> ;
+  acl:default <.> ;
+  acl:mode acl:Read .
+
+<#new-authorization>
+  a acl:Authorization ;
+  acl:agent ex:Carol ;
+  acl:default <.> ;
+  acl:mode acl:Read, acl:Write .
+```
+
+We could also use WAC's effective access control resource discovery mechanism and augment the content of `</weekly-status/.acl>`.
+
+```Turtle
+# Resource: </weekly-status/.acl>
+<#authorization>
+  a acl:Authorization ;
+  acl:agentGroup </groups/research#g1> ;
+  acl:default <.> ;
+  acl:mode acl:Read .
+
+<#new-authorization>
+  a acl:Authorization ;
+  acl:agent ex:Carol ;
+  acl:default <./2021-04-28/> ;
+  acl:mode acl:Read, acl:Write .
+```
```

Comment:
When `/weekly-status/.acl` is associated with `/weekly-status/` , `acl:default <./2021-04-28/>` will not be used towards evaluation (as per current WAC ED). This possibility was raised but needs to be vetted/implemented.

Reply:
I see [Tim's answer on issue 55](https://github.com/solid/specification/issues/55#issuecomment-536092174) clearly stating that `acl:Authorization` statements pointing via `acl:default` to a different object node from the URL of the resource for which the current ACL is defined should be ignored.

But I can't find the counterpart in the current WAC ED.
I checked [#authorization-conformance](https://solid.github.io/web-access-control-spec/#authorization-conformance), [#authorization-evaluation](https://solid.github.io/web-access-control-spec/#authorization-evaluation) & [#authorization-matching](https://solid.github.io/web-access-control-spec/#authorization-matching).

Do you know where I could find this?
Is this specific point worth raising as an issue on https://github.com/solid/web-access-control-spec?

Reply:
The definition in http://www.w3.org/ns/auth/acl#default dictates.

acl:default can only be a container. Perhaps https://solid.github.io/web-access-control-spec/#acl-default can be improved.

Earlier text used this https://github.com/solid/web-access-control-spec/pull/83/commits/173654fd4318078d2bcfac4ee95c2450aa17d18f#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L572 and it was slightly changed to express your concerns.. but (my) intention was to always keep it with the container that's in context (re determining effective ACL resource). If you feel that can be clarified, happy to update the text.

https://github.com/solid/authorization-panel/issues/191 is one issue that proposes to relax default.

Reply:
From the ontology:

> If a resource has no ACL file (it is 404), then access to the resource if given by the ACL of the immediately containing directory, or failing that (404) the ACL of the recursively next containing directory which has an ACL file. Within that ACL file, any Authentication which has that directory as its acl:default applies to the resource. (The highest directory must have an ACL file.)

It is a little confusing, but I get how that mandates not applying authorizations targetting subdirectories.

I think the current definition as it stands [in the WAC ED](https://solid.github.io/web-access-control-spec/#acl-default) is really good as is. But I don't see how it conveys this restriction. I think ultimately, I would agree with relaxing `acl:default` & `acl:accessTo` as described in https://github.com/solid/authorization-panel/issues/191.

I will add a note about all this rather than completely removing that section.